Put in tests for sampler (#248)
kokkos/kokkos-tools: README.md

The following provides an overview of the tools available in the set of Kokkos Tools. Click on each Kokkos Tools name to see more details about the tool via the Kokkos Tools Wiki. 

### Utilities
+ [**KernelFilter:**](https://github.com/kokkos/kokkos-tools/wiki/KernelFilter) 

    A tool which is used in conjunction with analysis tools, to restrict them to a subset of the application.

+ [**KernelSampler:**](https://github.com/kokkos/kokkos-tools/wiki/KernelSampler)

>    A tool which is used in conjunction with analysis tools, to restrict the tooling to samples of Kokkos kernel invocations.

Reviewer: ```suggestion
   A tool to be used in conjunction with analysis tools to restrict the tooling to samples of Kokkos kernel invocations.
```

Author: This is fine. I was trying to match the kernel filter description for now, but I would say the kernel filter utility wording needs to be fixed too. I think we can make changes on the other parts of the README as a  another PR.
